operator/charts/paladin-operator/values.yaml

```diff
@@ -1,15 +1,17 @@
 
 
-# Default values for paladin-operator
+# Install CRDs as part of the chart installation
+# When this is set to false, the CRDs are expected to be installed separately
+installCRDs: false
 
 # Default values for paladin-operator
 operator:
   name: paladin-operator
   namespace: paladin
   image:
-    repository: paladin-operator
-    tag: latest
-    pullPolicy: IfNotPresent
+    repository: kaleidoinc/paladin-operator
+    tag: release
+    pullPolicy: Always
```

Comment:
Think this is a nice add for now, but we'll likely need to come back to this as part of wider release management strategy as we go forwards.

Reply:
Definitely!! (same with the paladin image, that will be a nightmare)